jax/numpy/lax_numpy.py

```diff
@@ -33,6 +33,7 @@
 import warnings
 
 import numpy as np
+import numpy.core.numeric as _nx 
```

Comment:
`complexfloating` is already a module-level variable; no need for the additional import.